op-node/rollup/superchain.go

```diff
@@ -56,8 +56,10 @@ func LoadOPStackRollupConfig(chainID uint64) (*Config, error) {
 	}
 
 	var depositContractAddress common.Address
+	var systemConfigProxyAddress common.Address
 	if addrs, ok := superchain.Addresses[chainID]; ok {
 		depositContractAddress = common.Address(addrs.OptimismPortalProxy)
+		systemConfigProxyAddress = common.Address(addrs.SystemConfigProxy)
 	} else {
 		return nil, fmt.Errorf("unable to retrieve deposit contract address")
 	}
```

Comment:
I already found this confusing before. 
```suggestion
	addrs, ok := superchain.Addresses[chainID]
	if !ok {
		return nil, fmt.Errorf("unable to retrieve deposit contract address")
	}
	depositContractAddress := common.Address(addrs.OptimismPortalProxy)
	systemConfigProxyAddress := common.Address(addrs.SystemConfigProxy)
```
Or maybe not even set the address variables and directly use `addrs`, seem to only be used once each.

Reply:
Yep this is better. Will fix.

Reply:
[589b602](https://github.com/ethereum-optimism/optimism/pull/9585/commits/589b602cdfc56ebb6f22f64170d8b73d69c465e1)